examples/python/cloud-function-write-to-bigquery/main.py

```diff
@@ -0,0 +1,55 @@
+from google.cloud import bigquery
+import looker_sdk
+client = bigquery.Client()
+sdk = looker_sdk.init40()
+
+def main(request):
+  get_data_from_looker()
+  write_to_file()
+  load_to_bq()
+  return("Successfully loaded data from Looker to BigQuery")
+
+def get_data_from_looker(query_id=1):
+  query = sdk.run_query(
+    query_id=query_id,
+    result_format="csv",
+    limit= 5000
+  )  
+  print("Successfully retrieved data from Looker")
+  return query
+
+def write_to_file():
+  data = get_data_from_looker() 
+  ### Transform headers and data ("User Name" to become "User_Name")
+  cnt = 0
+  for i in data: 
+    if i == "\n":
+        break
+    else: 
+        cnt += 1
+  header = data[:cnt]
+  header_to_write = header.replace(" ", "_")
+  data_to_write = data[cnt:]
+  # Write header and data to temporary disk
+  with open('/tmp/table.csv', "w") as csv: # Files can only be modified/written inside tmp/
+    csv.write(header_to_write)
+    csv.write(data_to_write)
+  print("Successfully wrote data to a CSV file stored in temporary disk")
+
+def load_to_bq():
+    # Prepare the table inside BQ in advance. Additional logic can be written to create a table 
+    # Example: https://github.com/googleapis/python-bigquery/blob/35627d145a41d57768f19d4392ef235928e00f72/samples/create_table.py
+    table_id = "myproject.myschema.mytable" 
+    job_config = bigquery.LoadJobConfig(
+        source_format=bigquery.SourceFormat.CSV, skip_leading_rows=1, autodetect=True,
+    )
+    with open("/tmp/table1.csv", "rb") as source_file:
```

Comment:
what is `'table1.csv'` and how does that relate to `'table.csv'` on L34?

Reply:
This is typo from my end. They should be the same (write to table.csv and then read from table.csv) - I made change to the code.